packages/form-layout/test/form-layout-auto-responsive.test.js

```diff
@@ -140,4 +140,121 @@ describe('form-layout auto responsive', () => {
       }
     });
   });
+
+  describe('colspan', () => {
+    let container;
+    beforeEach(async () => {
+      container = fixtureSync(`
+        <div>
+          <vaadin-form-layout auto-responsive auto-rows>
+            <input placeholder="First name">
+            <input placeholder="Last Name">
+            <input placeholder="Email">
+            <input placeholder="Phone">
+            <input placeholder="Address" colspan="2">
+          </vaadin-form-layout>
+        </div>
+      `);
+      layout = container.firstElementChild;
+      await nextFrame();
+    });
+
+    it('should make element with colspan take the right amount of columns', () => {
+      expect(getComputedStyle(layout.children[4]).gridColumnEnd).to.equal('span 2');
+    });
+
+    it('should cap colspan if the layout get smaller', async () => {
```

Comment:
Let's also verify that the colspan returns to its original value when the container gets larger.

```suggestion
    it('should adjust colspan when container size changes, async () => {
```

Reply:
Please also update it to test that the colspan returns to its original value when the container size increases.

Reply:
Done.